package.json

```diff
@@ -49,21 +49,22 @@
   },
   "homepage": "https://github.com/fletcherist/yandex-dialogs-sdk#readme",
   "dependencies": {
-    "@types/node-fetch": "^2.1.2",
     "node-fetch": "^2.1.2"
   },
   "devDependencies": {
-    "@types/express": "^4.16.0",
+    "@types/debug": "^0.0.30",
+    "@types/node-fetch": "^2.1.2",
```

Comment:
Are you sure that node fetch types have to be moved here? I beliave that they should be located at same section as node-fetch

Reply:
thanks